charts/super-agent-deployment/templates/deployment-superagent.yaml

```diff
@@ -67,6 +68,16 @@ spec:
             - name: super-agent-config
               mountPath: /etc/newrelic-super-agent
               readOnly: true
+            {{- if .Values.config.fleet }}
+            {{- range $fleetName, $fleetConfig := .Values.config.fleet }}
+            {{- $chartSuffix := include "newrelic.common.naming.fullname" $ }}
+            {{- $configName := printf "%s-%s" "fleet" $fleetName }}
+            {{- $name := include "newrelic.common.naming.truncateToDNSWithSuffix" (dict "name" $chartSuffix "suffix" $configName) }}
+            - name: fleet-configmap-{{ $name }}
+              mountPath: /etc/newrelic-super-agent/fleet/agents.d/{{ $fleetName }}
+              readOnly: true
+            {{- end }}
+            {{- end }}
```

Comment:
would make sense to move this to a function instead? (unless we decide to go for a single cm)

Reply:
I would prefer to pair with CAOS tomorrow before making more changes.

I would like to convince them that the path they choose is a bad idea.